.eslintrc

```diff
@@ -1,15 +1,190 @@
 {
-  "parser": "babel-eslint",
-  "extends": "airbnb/base",
```

Comment:
Yea it seems like you handrolled airbnb for the most part there haha so it can stay :D